basic-ellipsis/public/index.html

```diff
@@ -1,65 +1,125 @@
-
-        try {
-          let app = firebase.app();
-          let features = ['auth', 'database', 'messaging', 'storage'].filter(feature => typeof app[feature] === 'function');
-          document.getElementById('load').innerHTML = `Firebase SDK loaded with ${features.join(', ')}`;
-        } catch (e) {
-          console.error(e);
-          document.getElementById('load').innerHTML = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
-    </script>
+        <header class="wrapper">
+          <div class="content_container content_container__without_border content_container__text_secondary_color">
+            <img src='images/Label.png' alt='label' class="header__logo" />
+            <h1 class="header__header">WORKING<br aria-hidden="true">WITH design LAB </h1>
+            <p class="header__text">
+              We’ve been helping startups launch and grow their businesses.
+              We help you to fill the gap between development, 
+                design and product management.
+            </p>
+            <img src='images/Mouse-icon.svg' alt='mouse image' class="header__mouse-logo" />
+          </div>
+        </header>
+        <div class="wrapper">
+          <div class="content_container">
+            <h2 class="content_container__header">About us</h2>
+            <p class="content_container__text">
+              When you work with us, you are part of a team of committed collaborators.
+              All aspects of our software development process are integrated, iterative and agile.
+               
+              Our approach unifies design, development and
+              product management to create exceptional products.
+              We start each project by understanding your business goals,
+              the user’s needs and the product requirements.
+              Then we help translate them into effective and elegant solutions.
+            </p>
+          </div>
+        </div>
+        <div class="wrapper wrapper__theme_services">
+          <div class="content_container content_container__services content_container__text_secondary_color content_container__border_secondary_color">
+            <h2 class="content_container__header content_container__services__header content_container__header__without_background"><span>Services</span></h2>
+            <p class="content_container__text">
+              Before we sign a contract or write a line of code our team will spend the necessary
+              time needed to understand your product vision.
+              The outcome of this scoping session is a proposal or high-level statement of work.
+              Based on your requirements, we can provide a complete offering or just the pieces
+              you need.
+            </p>
+          </div>
+        </div>
+        <div class="wrapper">
+          <div class="content_container">
+            <h2 class="content_container__header">What we do</h2>
+            <div class="group">
```

Comment:
Doesn't this one looks as like as content-container ?
(same stuff but without border and with image)